### PR TITLE
WIP - Orders panel part 5 (More styling and tweaking)

### PR DESCRIPTION
**Pull request by biocom**
# Summary

- Implemented some feedback and worked on styling
- Shows the 'kind' of order (SELL or BUY)
- Show the network card next to your ENSname or wallet address (@elena-zh )
- Misc other stuff.

# Todo next PR
- Further dig into transaction status labels and data
- Mobile responsive retouches.

https://user-images.githubusercontent.com/31534717/129058878-b7666e3c-4376-4ba4-b570-a81ae3f1932f.mov

https://user-images.githubusercontent.com/31534717/129058906-5bc10781-28dd-4846-b53f-e2f8817d90fd.mov

### CODE REVIEW

File: src/custom/components/AccountDetails/Transaction.tsx [hunk 1]
```diff
@@ -488,10 +497,10 @@ export default function Transaction({ hash: id }: { hash: string }) {
             <TransactionStatusText>
               {isCancelling ? (
                 <MouseoverTooltip text={activity.summary || id}>
-                  {getActivitySummary({ activityData, id })}
+                  {ActivitySummary({ activityData, id })}
```
Comment (alfetopito) [inline src/custom/components/AccountDetails/Transaction.tsx:500]: Even more component-y:

```suggestion
                  <ActivitySummary activityData={activityData} id={id} />
```

**Review by alfetopito — APPROVED** [review]
Working perfectly!

Next step, add the title with the full amount to lines 290 and 294: 
screenshot_2021-08-11_13-57-17

Like my full amounts title PR

**Comment by elena-zh** [comment]
PR changes look good to me. 
The only issue related to this PR - #1244

Also, I have created separate issues reported in #1227 comments and not fixed in the current pr: it will be a bit easier to track changes in the next related PRs, and not to forget about this or that issue: 
#1235 
#1236 
#1237 
#1238 
#1240 
#1241 
#1243 
#1242

**Review by elena-zh — APPROVED** [review]

**Comment by anxolin** [comment]
Nit: Nice to add something for the "empty list case":
image

I had an issue testing this in Rinkeby:
image

I do have balance:
image

Context:
https://gnosisinc.slack.com/archives/C025G521XQD/p1629119770004300

**Comment by elena-zh** [comment]
> Context:
> https://gnosisinc.slack.com/archives/C025G521XQD/p1629119770004300

and in #1262

**Comment by anxolin** [comment]
My issue had to do with one bug in the backend. They are looking into it. I will re-test once is solved. Or can i use production for doing the test tomorrow if it takes a bit to fix

**Comment by biocom** [comment]
Merging to continue the waterfall PR. @anxolin please review the 'no balance' issue in the final PR?

**Pull request merged**